### PR TITLE
!nit! Be consistent about `msbuild` quotes

### PR DESCRIPTION
**Pull request by dougbu**
- always quote comparison operands in `Condition`s

### CODE REVIEW

File: src/Servers/IIS/IIS/test/IIS.Tests/IIS.Tests.csproj [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
   <ItemGroup>
     <None Include="AppHostConfig\HostableWebCore.config" CopyToOutputDirectory="PreserveNewest" Link="%(FileName)%(Extension)" />
-    <None Include="$(AspNetCoreModuleV2ShimDll)" Condition="$(PackNativeAssets) == 'true'" CopyToOutputDirectory="PreserveNewest" Link="%(FileName)%(Extension)" />
+    <None Include="$(AspNetCoreModuleV2ShimDll)" Condition="'$(PackNativeAssets)'== 'true'" CopyToOutputDirectory="PreserveNewest" Link="%(FileName)%(Extension)" />
```
Comment (TanayParikh) [inline src/Servers/IIS/IIS/test/IIS.Tests/IIS.Tests.csproj:26]:
```suggestion
    <None Include="$(AspNetCoreModuleV2ShimDll)" Condition="'$(PackNativeAssets)' == 'true'" CopyToOutputDirectory="PreserveNewest" Link="%(FileName)%(Extension)" />
```

**Review by TanayParikh — APPROVED** [review]

**Review by dougbu — COMMENTED** [review]
typo

File: eng/targets/FunctionalTestWithAssets.targets [hunk 2]
```diff
@@ -6,7 +6,7 @@
   -->
   <Target Name="PublishTestAssets"
       BeforeTargets="Publish;RunTests;VSTest"
-      Condition=" '@(TestAssetProjectReference->Count())' != 0 ">
```
Comment (dougbu) [inline eng/targets/FunctionalTestWithAssets.targets:9]: (It's strange mixes like this that inspired cleaning up our operands.)

File: src/Components/WebAssembly/WebAssembly.Authentication/src/Microsoft.AspNetCore.Components.WebAssembly.Authentication.csproj [hunk 3]
```diff
@@ -76,7 +76,7 @@
       <_InteropBuildOutput Include="$(YarnWorkingDir)dist\$(Configuration)\**" Exclude="$(YarnWorkingDir)dist\.gitignore" />
     </ItemGroup>
 
-    <DefineStaticWebAssets Condition="@(_InteropBuildOutput) != ''"
+    <DefineStaticWebAssets 'Condition="'@(_InteropBuildOutput)' != ''"
```
Comment (dougbu) [inline src/Components/WebAssembly/WebAssembly.Authentication/src/Microsoft.AspNetCore.Components.WebAssembly.Authentication.csproj:79]:
```suggestion
    <DefineStaticWebAssets Condition="'@(_InteropBuildOutput)' != ''"
```

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**